CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# v2.0.0
```

Comment:
Two questions:

- Did you update the gem version? I can't see a version change in the code itself?
- Given we probably want to remove a bunch of other Heroku/Hirefile related stuff (see "NEXT" open PR) would it be worth using something like `2.0.0.pre1` so we can do a bunch of pre-releases before getting to the final 2.0.0 with everything gone? Otherwise each removal will be another major version, then we'll be up to a much bigger number very soon.